cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/TriggerTestCaseEventListener.java

```diff
@@ -77,6 +77,8 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
                         data.get(EventConstants.SCORE_ERROR_MSG));
                 break;
             case ScoreLangConstants.SLANG_EXECUTION_EXCEPTION:
+                flowFinished.set(true);
```

Comment:
Eliya just merged Shehab's change that ensures now that when we get to a SLANG_EXECUTION_EXCEPTION event, we will always get also a SCORE_ERROR_EVENT, so you can remove this line now